modules/social_features/social_follow_content/config/install/message.template.create_comment_following_node.yml

```diff
@@ -20,13 +20,13 @@ label: 'Create comment on following node'
 description: 'A person commented on content I am following'
 text:
   -
-    value: "<p><a href=\"[message:author:url:absolute]\">[message:author:display-name]</a> commented on [social_comment:parent_entity_author]'s <a href=\"[message:field_message_related_object:entity:url:absolute]\">[social_comment:commented_content_type]</a> you are following</p>\r\n"
+    value: "<p><a href=\"[message:author:url:absolute]\">[message:author:display-name]</a> commented on [social_comment:parent_entity_author]'s [social_comment:commented_content_type] <a href=\"[message:field_message_related_object:entity:url:absolute]\">[message:node-title]</a> you are following</p>\r\n"
```

Comment:
@jaapjan was it a deliberate decision not to make `[social_comment:parent_entity_author]` a link? I can imagine we didn't do this because it wasn't a link before, just checking.

Reply:
Just discussed with Xinyu that we should keep it as-is.